misc/build.func

```diff
@@ -129,6 +129,23 @@ ssh_check() {
   fi
 }
 
+generate_app_name() {
+    if ! command -v figlet &> /dev/null; then
+        sudo apt-get install -y figlet &> /dev/null
+    fi
+
+    # Use the Value of the APP Variable
+    APP="$1"
+
+    # Create and Formatting ASCII
```

Comment:
```suggestion
```

Reply:
I would leave it for documentation reasons. Source comments are always good

Reply:
self-explanatory code is always better, and I think `ascii_art=$(figlet -f slant "$APP")` is self-explanatory. But can leave the comment if you think it is important

Reply:
actually i think we should remove all comments in the function, and have a comment above function declaration `# This function`generates an ascii art app header`. This is in line with many other functions in `build.func`